### ISSUE TITLE
Placeholder files from dotnet/runtime packages cause warnings

### ISSUE THREAD

**Issue by Rabadash8820** [xoofx/UnityNuGet#182]
Please see the conversation over at dotnet/runtime#83710. Apparently, many dotnet/runtime packages (e.g., System.Reflection.Emit) include a placeholder `_._` file under different framework folders in the `lib` folder of the nupkg. This indicates that the package has nothing to add in those frameworks; all functionality is already present in the framework binaries. Unfortunately, UnityNuGet (or the Editor itself, I'm not sure which) is removing this placeholder file (maybe treating it as a [hidden asset](https://docs.unity3d.com/Manual/SpecialFolders.html)), but leaving a .meta file. This leads to Console warnings about .meta files with no associated folder. These warnings are quite distracting, seeming to appear every time the project is opened. I'm also concerned now that UnityNuGet is falling back to the wrong version of these NuGets, rather than referencing nothing like other NuGet clients would.

**Comment by Rabadash8820** [comment]
I'm not sure if this Issue has been addressed in the latest UnityNuGet versions, but if so then I think that the fix should be acknowledged by a developer here before this Issue is closed.

**Comment by Rabadash8820** [comment]
@bdovaz Has this issue actually been fixed?

**Comment by Rabadash8820** [comment]
Lol guess I'll just keep refreshing this Issue until I get a chance to actually test it myself

**Comment by bdovaz** [comment]
@Rabadash8820 create a PR to fix this if you want.

Thanks.

**Comment by Rabadash8820** [comment]
Okay, I just created a fresh Unity 2022.3.7f1 project with UnityNuGet and installed System.Reflection.Emit. This did not repro the warning that I described above, even after reimporting all assets and reopening the project (with and without deleting the `Library/` folder). Opening the package's contents under `Library/PackageCache/org.nuget.system.reflection.emit@4.7.0/`, I no longer see any empty directories or mismatched `.meta` files, so I guess this Issue is indeed resolved. 👍
